node/src/blockchain/blockchain_bridge.rs

```diff
@@ -51,13 +63,16 @@ impl Handler<BindMessage> for BlockchainBridge {
     type Result = ();
 
     fn handle(&mut self, msg: BindMessage, _ctx: &mut Self::Context) -> Self::Result {
-        self.set_consuming_wallet_subs_opt = Some(vec![
-            msg.peer_actors
-                .neighborhood
-                .set_consuming_wallet_sub
-                .clone(),
-            msg.peer_actors.proxy_server.set_consuming_wallet_sub,
-        ]);
+        //TODO is this dead code?
```

Comment:
I don't think it's precisely _dead_ code, but it's currently-unused code. I'd say comment it back in; we made a card to use it.